platform/linux-generic/odp_stash.c

```diff
@@ -57,8 +60,12 @@ typedef struct stash_t {
 typedef struct stash_global_t {
 	odp_ticketlock_t  lock;
 	odp_shm_t         shm;
-	uint8_t           stash_reserved[CONFIG_MAX_STASHES];
+	uint32_t          max_num;
+	uint32_t          max_num_obj;
+	uint32_t          num_internal;
+	uint8_t           stash_state[CONFIG_MAX_STASHES];
```

Comment:
`stash_state` could be part of `stash_t`, no extra space required if `index` type is changed and as stashes are no longer behind separate handles.

Reply:
Yeah, true, makes more sense to keep the state separate.